### PR TITLE
Add perigee_health_plots and Ska.report_ranges

### PR DESCRIPTION
**Pull request by jeanconn**
Add perigee_health_plots and Ska.report_ranges

### CODE REVIEW

**Comment by taldcroft** [comment]
Are either of this intended to be part of ska3-flight?  Ska.report_ranges seems like a candidate, though I'm not sure quite where it gets used, and don't really mind either way. Seems like perigee_health_plots is another one for ska-production #188?

It seems like it would be a good idea to generally also update other relevant metapackage(s) at the time of making a recipe, assuming the intent of the recipe is to get the package installed reproducibly.

**Comment by jeanconn** [comment]
I figured updating the metapackages was really a separate step compared to getting recipes to build conda packages to test first, as, for example, haven't figured out what versions go anywhere yet.  ska3-production looks like it would probably have fixed versions?

And yes, I figured Ska.report_ranges in ska3-flight and perigee_health_plots in ska3-production.

File: pkg_defs/Ska.report_ranges/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - chandra.time
+    - testr
+
+test:
+  imports:
+    - Ska.report_ranges
+
+
+about:
+  home: https://github.com/sot/aca_hi_bgd
```
Comment (jeanconn) [inline pkg_defs/Ska.report_ranges/meta.yaml:35]: Oops.

**Comment by taldcroft** [comment]
It needs to be done as a documented and reproducible process (as in copy/paste these lines and it will work) in the shiny maintainers Wiki.

The hint about where a package comes from is in /proj/sot/ska/www/ASPECT_ICXC/ska3-conda/shiny/repodata_*.json. Each package in there has a key `is_ska` that indicates if that package was built by ska_builder or is OTS. 

However, that may be different from flight / core, so in fact your original idea of building the core env, getting packages, and then adding in flight-latest seems fine. It just needs to be supplemented by tools / instructions that do not require any thinking or opening an editor.

**Comment by jeanconn** [comment]
I was asking in the context of this PR in which you seemed to be telling me to put Ska.report_ranges in ska3-flight.  It sounds like there will be no one-line edits to ska3-flight for a while and my adding Ska.report_ranges to ska3-flight-latest in this PR instead is probably the right thing.  *Then* some kind of process will be used to make ska3-flight.

**Comment by taldcroft** [comment]
> my adding Ska.report_ranges to ska3-flight-latest in this PR instead is probably the right thing. Then some kind of process will be used to make ska3-flight.

Agree on both.

**Pull request merged**